components/Text/text.css

```diff
@@ -0,0 +1,65 @@
+@import url('/utils.css');
+.heading-container{
+    display: flex;
+    flex-direction: column;
```

Comment:
why not use from a utility, when you have one?

Reply:
thanks for pointing out. Turns out it was of no use, so removed it.